package.json

```diff
@@ -138,7 +138,7 @@
     "@types/mocha": "^2.2.42",
     "@types/node": "^8.10.25",
     "@types/semver": "^5.5.0",
-    "typescript": "^2.6.1",
+    "typescript": "^3.2.2",
```

Comment:
That was a smooth upgrade 😆

Reply:
I'm surprised as well, the trick is that we use very little dependencies and mostly stable ones :D